app/views/user_mailer/_returned_overdue.html.erb

```diff
@@ -0,0 +1,4 @@
+<p>This is a notification for your late equipment checkin. </p>
```

Comment:
We should rewrite this to say "This is a receipt for your late equipment return." and go from there for cases when a receipt e-mail is requested. We should, however, only show the late fee statement if there is a late fee, so we'll need to add another conditional here (`<% if @reservation.equipment_model.late_fee > 0 %>` etc)